src/main/resources/studentEmailFragment-googleIdReset.html

```diff
@@ -1,32 +1,32 @@
 <p>
-    As requested, we have reset the link from your TEAMMATES account
-    for the course <em>${courseName}</em> to its previous Google account.
+  As requested, we have reset the link from your TEAMMATES account
+  for the course <em>${courseName}</em> to its previous Google account.
 </p>
 
 <p>
-    <strong>If you wish to continue to access TEAMMATES using your Google account for this course,
-        please rejoin this course through this Web address:
-    </strong>
-    <a href="${joinUrl}">${joinUrl}</a>
-    <br>*If prompted to log in, use your Google account to log in. If you do not have a Google account,
-    please create one from the
-    <a href="https://accounts.google.com/NewAccount">Google Accounts page</a>.
-    <br>*The above link is unique to you. Please do not share it with your classmates.
+  <strong>If you wish to continue to access TEAMMATES using your Google account for this course,
+    please rejoin this course through this Web address:
+  </strong>
+  <a href="${joinUrl}">${joinUrl}</a>
+  <br>*If prompted to log in, use your Google account to log in. If you do not have a Google account,
+  please create one from the
+  <a href="https://accounts.google.com/NewAccount">Google Accounts page</a>.
+  <br>*The above link is unique to you. Please do not share it with your classmates.
 </p>
 
 <p>
-    <strong>If you did not request for account reset before,
-        please contact our TEAMMATES support team at ${supportEmail}.
-    </strong>
+  <strong>If you did not request for account reset before,
```

Comment:
Line break after `<strong>`. Also, between `before, ` and `please` there may not need be a line break.